PusherChatkit.xcodeproj/project.pbxproj

```diff
@@ -788,11 +778,9 @@
 				SUPPORTED_PLATFORMS = "iphonesimulator iphoneos macosx appletvsimulator watchsimulator watchos appletvos";
 				SWIFT_VERSION = 4.2;
 				TARGETED_DEVICE_FAMILY = "1,2,3,4";
-				TVOS_DEPLOYMENT_TARGET = 10.0;
```

Comment:
Were those deletions of deployment targets intentional?

Reply:
This deletion is intentional. I moved these settings from target level to project level so that it is now consistent across all targets.